scripts/extract-l10n.sh

```diff
@@ -0,0 +1,166 @@
+#! /bin/bash
```

Comment:
I think this script can be removed, since we're going the route of the Django management commands instead and also using PyBabel's update command instead of `gettext`'s `msgmerge` utility?